docs/quickstarts/aws.md

```diff
@@ -27,7 +27,7 @@ By the end of this guide, you'll have a production-ready Kubernetes cluster runn
 * Basic understanding of Kubernetes concepts.
 * AWS account and IAM credentials.
 * AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct)
+* Terraform v0.12.x and [terraform-provider-ct v0.5.0](https://github.com/poseidon/terraform-provider-ct)
```

Comment:
Nit: I would move the `v0.5.0` out of the hyperlink.

Reply:
Done.